tests/wpunit/PostObjectCursorTest.php

```diff
@@ -16,7 +16,68 @@ public function setUp(): void {
 		$this->admin            = $this->factory()->user->create( [
 			'role' => 'administrator',
 		] );
+		$this->start_count      = 100;
+		$this->start_time       = time();
 		$this->created_post_ids = $this->create_posts();
+
+		register_graphql_fields(
```

Comment:
We shouldn't be doing this on all the tests, just `testPostOrderingByMultipleMeta()`

Reply:
Moved it.